client/src/components/Workbench/Workbench.js

```diff
@@ -9,30 +9,43 @@ import UIPatterns from '../UIPatterns';
 import Sidebar from '../Sidebar';
 import DataFiles from '../DataFiles';
 import History from '../History';
+import Onboarding from '../Onboarding';
 import * as ROUTES from '../../constants/routes';
 import NotificationToast from '../Toasts';
 import './Workbench.scss';
 
 function Workbench() {
   const { path } = useRouteMatch();
   const dispatch = useDispatch();
+  const demographics = useSelector(state => state.profile.data.demographics);
   // Show some entries only in local development
   const isDebug = useSelector(state =>
     state.workbench.status ? state.workbench.status.debug : false
   );
   const showUIPatterns = isDebug;
+
+  const setupComplete = demographics.setup_complete
```

Comment:
This feels wrong.  What about having a setupComplete flag in the initialState for demographics or whatever? then the useSelector hook will pick up the initial "false" value and then change once the dispatch and saga complete.

Reply:
Yes, that sounds cleaner. Will do 👍

Reply:
>setupComplete flag in the initialState for demographics or whatever?
for location of setupComplete?  why not in `/api/workbench`?

(I just used profile.demographics because it was already there.  but that route seems to not match up with our use as it also calls pytas and has a different goal.)